Makefile

```diff
@@ -1,4 +1,4 @@
-TAG="latest"
+TAG=$(shell cat build.properties | cut -d'=' -f 2)
```

Comment:
SNAPSHOT will be there in the version. Is it fine?

Reply:
IMO, yes.  Our versions/tags should have -SNAPSHOT or some other string to indicate that the build isn't an official release.  We will need to make sure release builds do not have -SNAPSHOT on them.

Reply:
That will automatically take care by Jenkins CI job

Reply:
That's what I has hoping for. :)

Reply:
ok. you can merge this.. Will change the cloud build accordingly then..